src/Search.php

```diff
@@ -964,8 +975,6 @@ public function view() {
 	public function view_paged_posts() {
 		// TODO - The $paged_context related code should be transferred to set_results_context method for better separation of concerns.
 		if ( $this->paged_posts ) {
-			$paged_context['settings'] = get_option( 'planet4_options' );
```

Comment:
Is this change is intentional?

Reply:
Yes, it is, because it seemed to be used only in [this line](https://github.com/greenpeace/planet4-master-theme/blob/main/templates/tease-search.twig#L144) that was removed in the PR but please let me know if I'm missing something!

Reply:
Nope, all good 👍